ReadMe.md

```diff
@@ -30,57 +25,82 @@ The Aiko Engine MP provides ...
 - Supports [LCA2017 IoTuz project](http://www.openhardwareconf.org/wiki/OHC2017)
 - Tested on a range of ESP32 development boards
 
-<a name="installation" />
+## Installation
+
+Installation is a three stage process:
+
+1. Setup the correct tooling on your workstation
+2. Download and install MicroPython onto your ESP32 development board
+3. Install the Aiko software onto your ESP32 development board
 
-Installation
--------------
+### SETUP
+
+- Make sure you have a current version of [Python3](https://www.python.org/) available on your workstation
+
+- Download the Aiko software code from https://github.com/geekscape/aiko_engine_mp
 
-- Download the source code from <https://github.com/geekscape/aiko_engine_mp>
 ```
     git clone https://github.com/geekscape/aiko_engine_mp
     cd aiko_engine_mp
 ```
-- Ensure you have a
-  [compatible hardware board](https://github.com/micropython/micropython/wiki/Boards-Summary) or
-  [compatible microPython port](https://github.com/micropython/micropython/tree/master/ports)
-- Make sure you have a Python
-  [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/#lower-level-virtualenv) set-up, including
-  [virtualenvwrapper](http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper)
-- Create `mkvirtualenv micropython` and work on the new virtual environment
-```
-    workon micropython
-```
-- Install mpfshell
+
+  Note: If you don't have Git installed you download a zip file from https://github.com/geekscape/aiko_engine_mp
+
+- Setup a [virtual Python environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) for your badge project
```

Comment:
Strikes me we could be a little more prescriptive here? Experienced Pythonistas will know how to do it their preferred way. For beginners we could add a `requirements.txt`, append `.venv` to `.gitignore`, and suggest they paste:

```bash
# the first time
python3 -m venv .venv
pip3 install -r requirements.txt
# once per new shell window or tab
export VIRTUAL_ENV="$PWD/.venv"
export PATH="$PWD/.venv/bin:$PATH
```

… which works on `bash` and hopefully also `zsh`, now the default shell on macOS. Perhaps there's a way to make it simpler? `direnv` would make it easier to automate setting the environment variables when you come back next time, but that's just adding more to their TODO list. :/

Reply:
God suggestions.

Might skip the direnv stuff

Reply:
Added